src/app/components/NavigationLink/NavigationLink.tsx

```diff
@@ -12,13 +11,13 @@ export function NavigationLink({
   children,
   onClick,
 }: NavigationLinkProps) {
-  const pathname = usePathname(); // Use usePathname hook to get the current path
+  const pathname = usePathname();
   const isActive = pathname === href;
   return (
-    <Link href={href} passHref className="usa-nav__link" onClick={onClick}>
+    <Link href={href} passHref onClick={onClick}>
       <span
-        className={classNames(styles.navbarItemText, {
-          'navbar-item-active': isActive,
+        className={classNames('text-white', {
+          underline: isActive,
```

Comment:
This is the main change so that we get the underlining. Otherwise, I just cleaned up some styling we didn't need.